Makefile

```diff
@@ -1,7 +1,43 @@
+ROOT:=$(shell git rev-parse --show-toplevel)
+COMMIT:=$(shell git rev-parse --short HEAD)
+MIN_RUNS:=25
```

Comment:
To decide this value, I started with 10 runs,  and then checked 50 runs. the difference in measurements were not that much, even the std dev hadn't changed that much from 10 runs to 50. So I settled on 25.